x-pack/legacy/plugins/siem/public/components/alerts_viewer/histogram_configs.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as i18n from './translations';
+import { MatrixHistogramOption } from '../matrix_histogram/types';
+import { HistogramType } from '../../graphql/types';
+
+export const alertsStackByOptions: MatrixHistogramOption[] = [
+  {
+    text: 'event.category',
+    value: 'event.category',
+  },
+  {
+    text: 'event.module',
+    value: 'event.module',
+  },
+];
+
+export const histogramConfigs = {
```

Comment:
If an interface defines the shape of a valid histogram config, consider referencing it here, because the type checker can do some compile-time validation of the initial values for us. 😄